app/src/main/res/layout/activity_request_off.xml

```diff
@@ -136,80 +138,132 @@
+                    <ImageView
+                        tools:ignore="ContentDescription"
+                        style="@style/StyleImageDropDown"
                         />
+                </FrameLayout>
 
-                    <TextView
-                        android:layout_width="match_parent"
-                        android:layout_height="50dp"
-                        android:layout_weight="1"
-                        android:background="@color/color_light_teal"
-                        android:gravity="center"
-                        android:text="@string/off_no_salary"
-                        android:textAllCaps="false"
-                        android:textColor="@android:color/white"
-                        />
-                </LinearLayout>
+                <FrameLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
 
-                <!-- Start layout of Off have salary-->
+                    android:visibility="@{viewModel.salaryFlag ? View.GONE : View.VISIBLE}"
 
-                <LinearLayout
-                    android:layout_width="match_parent"
-                    android:layout_height="@dimen/dp_50"
-                    android:orientation="horizontal"
-                    android:padding="@dimen/dp_5"
-                    android:weightSum="2"
-                    >
 
-                    <Button
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginEnd="@dimen/dp_5"
-                        android:layout_marginRight="@dimen/dp_5"
-                        android:layout_weight="1"
-                        android:background="@color/colorPrimary"
-                        android:text="@string/company_pay"
-                        android:textAllCaps="false"
-                        android:textColor="@android:color/white"
-                        tools:ignore="ButtonStyle"
-                        />
 
-                    <Button
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginLeft="@dimen/dp_5"
-                        android:layout_marginStart="@dimen/dp_5"
-                        android:layout_weight="1"
-                        android:background="@color/color_light_teal"
-                        android:text="@string/insurance_coverage"
-                        android:textAllCaps="false"
-                        android:textColor="@android:color/white"
-                        tools:ignore="ButtonStyle"
+                    >
+                    <android.support.design.widget.TextInputLayout
+                        style="@style/StyleTextHint"
+                        >
+                        <EditText
+                            android:hint="Off have salary type"
```

Comment:
WARNING: [HardcodedText] [I18N] Hardcoded string "Off have salary type", should use `@string` resource
 Hardcoding text attributes directly in layout files is bad for several reasons:

* When creating configuration variations (for example for landscape or portrait)you have to repeat the actual text (and keep it up to date when making changes)

* The application cannot be translated to other languages by just adding new translations for existing string resources.

There are quickfixes to automatically extract this hardcoded string into a resource lookup.